Ds3Examples/Ds3ExampleClient.cs

```diff
@@ -404,10 +409,10 @@ static void Main(string[] args)
                 exampleClient.runPut(bucket, testSourceDirectory, testSourceFile);
 
                 // put a single file into the bucket with precomputed checksum
-                exampleClient.runPutWithChecksum(bucket, testSourceDirectory, testChecksumFile, testChecksumCrc32C, Checksum.ChecksumType.Crc32C);
+                exampleClient.runPutWithChecksum(bucket, testSourceDirectory, testChecksumFile, testChecksumCrc32C, ChecksumType.Type.CRC_32C);
```

Comment:
I would comment this out and explain that if the data policy is configured to accept CRC_32C hashes, this is how you would do it.